code/game/objects/items/weapons/twohanded.dm

```diff
@@ -159,6 +159,9 @@
 
 /obj/item/weapon/twohanded/required/equipped(mob/user, slot)
 	..()
+	var/slotbit = slotdefine2slotbit(slot)
```

Comment:
Short of making the slots like slot_back equal to the binary defines I see no other way to do it. And I saw the comment when I found the proc, good to know who it belongs to.